### PR TITLE
RK-1717 - java-aws-elasticbeanstalk tests

### PR DESCRIPTION
**Pull request by ManuelNemirovsky**

### CODE REVIEW

File: java-aws-elasticbeanstalk/runTest.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+aws s3 cp /app/java-elastic-beanstalk.zip s3://java-elastic-beanstalk-regression-test ; \
+
+now="$(date +'%d-%m-%Y-%H:%M:%S')" ; \
```
Comment (MickaelAlliel) [inline java-aws-elasticbeanstalk/runTest.sh]: remove all ; \

File: java-tomcat-aws-elasticbeanstalk/README.md [hunk 2]
```diff
@@ -72,16 +65,7 @@ We have added Rookout to the original project by:
...
-            command: wget "https://get.rookout.com" -O setup.sh
-        "02": 
-            command: sudo /bin/bash setup.sh agent --token=<YOUR_TOKEN>
-        "03": 
-            command: /etc/init.d/rookout-agent start
-    ```
+2. Adding Rookout's Elastic Beanstalk .ebextensions to install the Rookout ETL Agent and add the Java Agent to communicate with the app: command: /etc/init.d/rookout-agent start
     ```
     files:
         "/opt/elasticbeanstalk/lib/rook.jar" :
```
Comment (MickaelAlliel) [inline java-tomcat-aws-elasticbeanstalk/README.md]: add rookout token instruction

File: java-aws-elasticbeanstalk/runTest.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+aws elasticbeanstalk create-application-version \
+                                --application-name regression-tests \
+                                --version-label $now \
+                                --source-bundle S3Bucket="java-elastic-beanstalk-regression-test",S3Key="java-elastic-beanstalk.zip" ; \
+
+aws elasticbeanstalk update-environment \
+                            --application-name regression-tests \
+                            --environment-name java-aws-elasticbeanstalk \
+                            --version-label $now \
+                            --option-settings Namespace=aws:elasticbeanstalk:application:environment,OptionName=ROOKOUT_TOKEN,Value=$ROOKOUT_TOKEN,OptionName=ROOKOUT_ROOK_TAGS,Value="http://java-aws-elasticbeanstalk",OptionName=ROOKOUT_DEBUG,Value=1
```
Comment (MickaelAlliel) [inline java-aws-elasticbeanstalk/runTest.sh]: remove http:// from tag

File: java-tomcat-aws-elasticbeanstalk/README.md (hunk 2 above)
Comment (ManuelNemirovsky) [reply java-tomcat-aws-elasticbeanstalk/README.md]: Fixed

**Pull request merged**